pkg/oc/generate/cmd/newapp.go

```diff
@@ -518,7 +522,13 @@ func (c *AppConfig) buildPipelines(components app.ComponentReferences, environme
 }
 
 // buildTemplates converts a set of resolved, valid references into references to template objects.
-func (c *AppConfig) buildTemplates(components app.ComponentReferences, parameters app.Environment, environment app.Environment, buildEnvironment app.Environment, templateProcessor templateinternalclient.TemplateProcessorInterface) (string, []runtime.Object, error) {
+func (c *AppConfig) buildTemplates(
+	components app.ComponentReferences,
+	parameters app.Environment,
+	environment app.Environment,
+	buildEnvironment app.Environment,
+	templateProcessor templateinternalclient.TemplateProcessorInterface,
+) (string, []runtime.Object, error) {
```

Comment:
please try to avoid inadvertent formatting changes in PRs (especially when it's the only change in the file).